src/main/java/org/apache/commons/collections4/MapUtils.java

```diff
@@ -1186,11 +1188,12 @@ public static void debugPrint(final PrintStream out, final Object label, final M
      *
      * @param <K> the key type
      * @param <V> the value type
-     * @param map the map to invert, may not be null
+     * @param map the map to invert, must not be null
      * @return a new HashMap containing the inverted data
      * @throws NullPointerException if the map is null
      */
     public static <K, V> Map<V, K> invertMap(final Map<K, V> map) {
+        Objects.requireNonNull(map, "The map must not be null");
```

Comment:
Please simply use the argument name like "map" or whatever the faulty argument name is.

Reply:
Thanks for your advice. I have updated it.